oci/tomcat/image.yaml

```diff
@@ -0,0 +1,36 @@
+version: 1
+
+upload:
+  - source: canonical/chiselled-tomcat
+    commit: 59f3f98b37988ab3a07419098e755c892c44f93d
+    directory: tomcat/9-jre-21-24.04
+    release:
+      9-21-24.04:
+        end-of-life: "2032-01-30T00:00:00Z"
+        risks:
+          - stable
+          - candidate
+          - beta
+          - edge
+  - source: canonical/chiselled-tomcat
+    commit: 59f3f98b37988ab3a07419098e755c892c44f93d
+    directory: tomcat/10-jre-21-24.04
+    release:
+      10-21-24.04:
```

Comment:
IMO the channel is still weird.

Our convention is `<swVersion>-<base>_<risk>`. In this case, I find it really hard for a user to understand what's actually being delivered. Furthermore, it will make it really hard for things like the store to filter the rock by "version"...because - _what is the version??_

I ack the challenge though. No perfect, but what about `10jre21-24.04`? I'd love to get the store folks opinion on this as well.

Reply:
@vpa1977 just checked with the store team, and the preferred approach would be:

`10-jre21-24.04` (`<swVersion>(-<platform>)-<base>`).

One other thing you can do, to preserve an intuitive UX for users, is to **still have** `10-24.04`, which uses the default jre in 24.04, and is the same artifact as `10-jre21-24.04`.